internal/schema/metadatasecrets.go

```diff
@@ -0,0 +1,59 @@
+package schema
+
+import (
+	"fmt"
+	"strings"
+
+	"google.golang.org/protobuf/proto"
+
+	schemapb "github.com/TBD54566975/ftl/backend/protos/xyz/block/ftl/v1/schema"
+)
+
+// MetadataSecrets represents a metadata block with a list of config items that are used.
+//
+//protobuf:11,optional
+type MetadataSecrets struct {
+	Pos Position `parser:"" protobuf:"1,optional"`
+
+	Secrets []*Ref `parser:"'+' 'secrets' @@ (',' @@)*" protobuf:"2"`
+}
+
+var _ Metadata = (*MetadataSecrets)(nil)
+
+func (m *MetadataSecrets) Position() Position { return m.Pos }
+func (m *MetadataSecrets) String() string {
+	out := &strings.Builder{}
+	fmt.Fprint(out, "+secrets ")
```

Comment:
I forgot to say, there are tests that should be updated to exercise this parsing, in particular TestParserRoundtrip et al and TestProtoRoundTrip.